lazyflow/utility/io_util/OMEZarrStore.py

```diff
@@ -41,6 +42,60 @@
 OME_ZARR_V_0_4_KWARGS = dict(dimension_separator="/", normalize_keys=False)
 OME_ZARR_V_0_1_KWARGS = dict(dimension_separator=".")
 
+# {
+#   "type": "scale" OR "translation",
+#   "scale": List[float] OR "translation": List[float] OR "path": str
+# }
+OMEZarrCoordinateTransformation = Dict[str, Union[str, List[float]]]
+
+
+def _remove_transforms_with_path(
+    coordinate_transformations: Optional[List[OMEZarrCoordinateTransformation]],
+) -> Optional[List[OMEZarrCoordinateTransformation]]:
+    """
+    coordinateTransformations may provide a path to transformation data in binary format, instead
+    of specifying floats for each axis. We don't know of any commonly used tool that writes such data,
+    so there is no way to know what to do with this.
```

Comment:
Bad docstring; the actual problem is that the spec doesn't actually allow including coordinateTransforms with a `"path"`, the same as it does not allow including an `identity` transformation (despite the coordinateTransformation spec defining it as a valid transformation). Quote [multiscale spec](https://ngff.openmicroscopy.org/latest/index.html#multiscale-md):

> The transformation MUST only be of type translation or scale. They MUST contain exactly one scale transformation that specifies the pixel size in physical units or time duration. If scaling information is not available or applicable for one of the axes, the value MUST express the scaling factor between the current resolution and the first resolution for the given axis, defaulting to 1.0 if there is no downsampling along the axis. It MAY contain exactly one translation that specifies the offset from the origin in physical units. If translation is given it MUST be listed after scale to ensure that it is given in physical coordinates. The length of the scale and translation array MUST be the same as the length of "axes".

Given that the scale transformation MUST be present for each `dataset`, and it MUST specify "the pixel size in physical units or time duration", I think a writer would be pretty misguided to think it can somehow create an OME-Zarr store that anyone else can read if it provides the scaling in some arbitrary binary format.

tl;dr I see 0 value spending any more time than changing the docstring on this